README.md

```diff
@@ -12,12 +12,15 @@ Material Components for the web is the successor to [Material Design Lite](https
 - Production-ready components consumable in an a-la-carte fashion
 - Best-in-class performance and adherence to the [Material Design guidelines](https://material.io/guidelines)
 - Seamless integration with other JS frameworks and libraries
-  - [Preact Material Components](https://github.com/prateekbh/preact-material-components)
-  - [RMWC: React Material Web Components](https://github.com/jamesmfriedman/rmwc)
-  - [Angular MDC](https://github.com/trimox/angular-mdc-web)
-  - [Blox Material](https://blox.src.zone/material): Angular Integration Library.
-  - [Vue MDC Adapter](https://github.com/stasson/vue-mdc-adapter): MDC Web Integration for Vue.js (using [foundation/adapters](./docs/integrating-into-frameworks.md#the-advanced-approach-using-foundations-and-adapters).)
-  - [Material Components Vue](https://github.com/matsp/material-components-vue): MDC Web Integration for Vue.js (using [vanilla components](./docs/integrating-into-frameworks.md#the-simple-approach-wrapping-mdc-web-vanilla-components))
+  - [Material Components for React](https://github.com/material-components/material-components-web-react): MDC Web Integration for React.js (using [foundation/adapters](./docs/integrating-into-frameworks.md#the-advanced-approach-using-foundations-and-adapters))
```

Comment:
* Drop ".js" from React? (It's referred to simply as React on their own site)
* lowercase i in integration
* foundation -> foundations